SpatialGDK/Documentation/content/toolbar.md

```diff
@@ -1,72 +1,90 @@
+You can launch multiple servers at the same time from within the Unreal Editor in [PIE (Unreal documentation)](https://docs.unrealengine.com/en-us/Engine/UI/LevelEditor/InEditorTesting#playineditor) configuration. To configure the number of servers launched, open the **Play** drop-down menu and use the slider `Number of Servers` within the `Multiplayer Options` section.
 
-* **Stop**</br>
-Stops `spatial local launch`.
+## SpatialOS GDK for Unreal toolbar
 
-* **Inspector**</br>
-Opens the [Inspector](https://docs.improbable.io/reference/latest/shared/glossary#inspector) (SpatialOS documentation) in a browser.
+The GDK toolbar provides several functions required for building and launching your client and server-workers from inside the Unreal Editor.
 
-## Settings
+### Add the GDK toolbar to your Unreal project
 
-The toolbar settings are in **Edit** > **Project Settings** > **SpatialOS Unreal GDK** > **Toolbar**.
+> Note: If you based your project off the [StarterProject](https://github.com/spatialos/UnrealGDKStarterProject), the toolbar is already enabled.
 
- ![Toolbar]({{assetRoot}}assets/screen-grabs/toolbar/toolbar-settings.png)
+To enable the GDK toolbar, navigate to **Edit** > **Plugins** inside the Unreal Editor and scroll down to the bottom. Select the **SpatialOS Unreal GDK** section and enable the toolbar:
 
-* **Configuration**
+![Toolbar]({{assetRoot}}assets/screen-grabs/toolbar/enable-toolbar.png)
+
+#### Buttons
+
+The GDK toolbar has five features mapped to individual buttons and is displayed in the main editor toolbar to the right of the `Launch` button:
+
+ ![Toolbar]({{assetRoot}}assets/screen-grabs/toolbar/toolbar-buttons.png)
 
-    * **Project root folder**</br>
-    If you're using a non-standard structure, you'll need to set this yourself. This is empty by default. If you leave it empty, it defaults to `<GameRoot>/../spatial`.
+You can also access these from the **Window** menu:
+
+ ![Toolbar]({{assetRoot}}assets/screen-grabs/toolbar/window-access.png)
 
-    * **Launch configuration**</br>
-    The [launch configuration file](https://docs.improbable.io/reference/latest/shared/reference/file-formats/launch-config) (SpatialOS documentation) to use when running `spatial local launch` using the `Launch` button.
+| Button | Description |
+| --- | --- |
+| Snapshot | Generates a [SpatialOS snapshot]({{urlRoot}}/content/glossary#snapshot). | 
+| Schema | Creates [schema]({{urlRoot}}/content/glossary#schema) for your Unreal project. |
+| Launch | Runs [`spatial worker build build-config` (SpatialOS documentation)](https://docs.improbable.io/reference/latest/shared/spatial-cli/spatial-worker-build-build-config) to build worker configs and runs `spatial local launch` with the launch configuration specified in the settings (see [below](#settings)). |
```

Comment:
@anne-edwards is this present in the glossary too?

Reply:
"Launch" is, but I don't think it's relevant to include a link here. `spatial worker build` and `spatial local launch` aren't. 

"launch configuration" is, but I don't know if it's worth linking out to the glossary, since we already link to the "Launch configuration" entry below, under Schema Generation, from where we link to the glossary entry.